game.py

```diff
@@ -116,4 +106,44 @@ def is_word_guessed(snowman_word, correct_letter_guess_statuses):
     for letter in snowman_word:
         if not correct_letter_guess_statuses[letter]:
             return False
-    return True
\ No newline at end of file
+    return True
+
+
+
+def snowman(snowman_word):
+    """Complete the snowman function
+    replace "pass" below with your own code
+    It should print 'Congratulations, you win!'
+    If the player wins and, 
+    'Sorry, you lose! The word was {snowman_word}' if the player loses
+    """
+
+    # This print statement is only for personal testing and   
+    # should be removed before sharing the code with others
+    print(f"debug info: {snowman_word}")
+    
+    # Add a new list to track correctly guessed letters
+    
+    wrong_guesses_list = []
```

Comment:
```suggestion
    # Add a new list to track correctly guessed letters
    wrong_guesses_list = []
```

We shouldn't leave blank lines between a comment and the code that it belongs to. It's easy for folks to get confused about what the comment is referring to, which often leads to losing time by needing to track folks down to ensure what the comment's original intention was.

Reply:
Yes, I will do it better next time.